__tests__/Middlewares/api-limiter.spec.js

```diff
@@ -0,0 +1,10 @@
+const request = require('super-test');
+const server = require('../../api/server');
+
+describe('Should be able to limit and control api data for users.', () => {
+
+    const user = {
+        sub: "github|26471655",
```

Comment:
is this a hardcoded id?

Reply:
this was a constructed user for a test that was being written. I'm going to leave it for now because if we get time we'll write the complete test.

Reply:
great! 👍